From 213f3a62663356657c1cd34996fed9a396996276 Mon Sep 17 00:00:00 2001
From: Buildbot <buildbot@xenial.com>
Date: Thu, 1 Sep 2022 10:42:52 +0000
Subject: [PATCH] Remove Python version check from cmake/Dependencies.cmake
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Yocto (any non-EOL version) is way past Python version 3.0.

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Inappropriate [Yocto specific]
---
 cmake/Dependencies.cmake | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index 44a8bf1211c..a24b99c6923 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -1053,15 +1053,6 @@ if(BUILD_PYTHON)
   find_package(PythonInterp 3.0)
   find_package(PythonLibs 3.0)
 
-  if(${PYTHONLIBS_VERSION_STRING} VERSION_LESS 3)
-    message(FATAL_ERROR
-      "Found Python libraries version ${PYTHONLIBS_VERSION_STRING}. Python 2 has reached end-of-life and is no longer supported by PyTorch.")
-  endif()
-  if(${PYTHONLIBS_VERSION_STRING} VERSION_LESS 3.7)
-    message(FATAL_ERROR
-      "Found Python libraries version ${PYTHONLIBS_VERSION_STRING}. Python 3.6 is no longer supported by PyTorch.")
-  endif()
-
   # When building pytorch, we pass this in directly from setup.py, and
   # don't want to overwrite it because we trust python more than cmake
   if(NUMPY_INCLUDE_DIR)
-- 
2.31.1

